painters/config/adaptive.yaml

```diff
@@ -0,0 +1,56 @@
+description: cnn-painter-by-number-entrypoint
+checkpoint_storage:
+  type: shared_fs
+  host_path: "/home/chris/checkpoints/painters"
+
+bind_mounts:
+  - host_path: "/home/chris/painters/data_all"
+    container_path: "/home/chris/painters/data_all"
+  - host_path: "/tmp"
+    container_path: "/tmp"
```

Comment:
Remove the `/tmp` directory if it isn't used in the model definition anymore, same below.

Reply:
Could you remind me of the use of line 9 and 10? Sorry I lost track of it.

Reply:
We were using this with our experimental profiling feature (`env.debug: 0.1.0`), undocumented.

Reply:
got it, thanks!